src/components/NewLocationPage/NotesBlock/NotesBlock.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+
+export const SectionContentContainer = styled.div`
+  display: flex;
+  font-size: initial;
+  text-transform: none;
+`;
+
+export const TextContainer = styled.div``;
+
+export const TextComponent = styled.p`
+  color: ${COLOR_MAP.GREY_4};
+  font-weight: initial;
+`;
```

Comment:
When I don't specify margin, the space between the paragraph and the title seems bigger than in the mock. Is it worthwhile to explicitly specify the top margin here or setting it to `initial`? (Although when I tried setting it to initial, it makes the space just a tiny bit too narrow).

Reply:
yeah, paragraph elements come with default margins. i would just override them and set the margin top to whatever the amount of space there is between title+body in the mocks. so something like `margin: .5rem 0 0;` . this sets all margins to be 0 except for the top-margin which is .5rem